### PR TITLE
Added 1.2.10

### PR DESCRIPTION
**Pull request by hexonaut**

### CODE REVIEW

File: releases/kovan/1.2.10/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Kovan Release 1.2.10</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/kovan/1.2.10/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/Qmc3VkuiwWNyPsdR3xzmwSs5aM24ff2jwGbjHPMS6Yt7dh">Contract addresses (IPFS hosted)</a></li>
```
Comment (zdumitru) [inline releases/kovan/1.2.10/index.html:13]: @brianmcmichael This doesn't look similar to Kovan contract addresses. For example, ETH-C is missing

File: releases/mainnet/1.2.10/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+				<strong>Mainnet Release 1.2.10</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.10/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmQBFYzikwVfwH1nE92eKtjX9xRFjTRG8rqbWBAg2STG9k">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.2.10/abi/index.html">ABIs</a></li>
+				 <li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.10" target="_blank">changelog</a>)</li>
```
Comment (zdumitru) [inline releases/mainnet/1.2.10/index.html:17]: Release notes are not ready yet. Either comment out or wait for dss-deploy-scripts release to be added to GitHub.

**Review by zdumitru — CHANGES_REQUESTED** [review]

File: releases/kovan/1.2.10/index.html (hunk 1 above)
Comment (brianmcmichael) [reply releases/kovan/1.2.10/index.html]: Old pin got moved up. Fixed.

File: releases/mainnet/1.2.10/index.html (hunk 2 above)
Comment (brianmcmichael) [reply releases/mainnet/1.2.10/index.html:17]: I've commented this out.

**Review by zdumitru — APPROVED** [review]

**Pull request merged**